src/ontology/imports/foodon_product_import.owl

```diff
@@ -33368,6 +33368,52 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004142 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004142">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00002951"/>
+        <obo:IAO_0000115 xml:lang="en">A patty made from rabbit meat.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date xml:lang="en">2021-12-20T15:39:21Z</dc:date>
+        <rdfs:label xml:lang="en">rabbit patty</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004143 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004143">
```

Comment:
definition?

Reply:
We already have 'patty' defined in foodOn, so I thought maybe we should keep its definition bit generalized.

Reply:
I had no problem with rabbit patty!